### PR TITLE
*: add traditional Chinese translation

### PR DESCRIPTION
**Pull request by Tylerastro**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.zh_TW/common/alias.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# alias
+
+> 建立快捷鍵名稱 -- 取代命令字串的單字
+> 快捷鍵只會在現有的shell有效，除非在shell的設定檔中定義快捷鍵，例如: `~/.bashrc`
+> 更多資訊: <https://tldp.org/LDP/abs/html/aliases.html>.
```
Comment (blueskyson) [inline pages.zh_TW/common/alias.md:5]:
```suggestion
> 快捷鍵只會在現有的 shell 有效，除非在 shell 的設定檔中定義快捷鍵，例如：`~/.bashrc`
> 更多資訊：<https://tldp.org/LDP/abs/html/aliases.html>.
```

File: pages.zh_TW/common/alias.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+- 查看與給定名稱有關的快捷鍵:
+
+`alias {{快捷名稱}}`
+
+- 移除快捷鍵:
+
+`unalias {{快捷名稱}}`
+
+- 將rm變成互動式指令:
```
Comment (blueskyson) [inline pages.zh_TW/common/alias.md:23]:
```suggestion
- 將 `rm` 變成互動式指令：
```

File: pages.zh_TW/common/alias.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+`alias {{rm}}="{{rm --interactive}}"`
+
+- 建立`la`作為`ls --all`的捷徑:
```
Comment (blueskyson) [inline pages.zh_TW/common/alias.md:27]:
```suggestion
- 建立 `la` 作為 `ls --all` 的捷徑：
```

File: pages.zh_TW/common/aws.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
+# aws
+
+> Amazon Web Services 官方的命令列介面工具。
+> 子命令（例如`aws s3`）有自己的說明文件。
```
Comment (blueskyson) [inline pages.zh_TW/common/aws.md:4]: Refer to [subcommand-mention](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/subcommand-mention.md#zh_tw).
```suggestion
> 此命令也有關於其子命令的文件，例如：`aws s3`.
```

File: pages.zh_TW/common/chmod.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+# chmod
+
+> 修改文件或目錄的存取權限
+> 使用者類型分為檔案所有者(u), 檔案所有者之群組(g), 以及其他使用者(o)
```
Comment (blueskyson) [inline pages.zh_TW/common/chmod.md:4]:
```suggestion
> 使用者類型分為檔案所有者 (u)、檔案所有者之群組 (g)、以及其他使用者 (o)。
```

File: pages.zh_TW/common/crontab.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+# crontab
+
+> 排程作業按時間間隔執行.
+> 更多資訊: <https://crontab.guru/>.
```
Comment (blueskyson) [inline pages.zh_TW/common/crontab.md:4]:
```suggestion
> 排程作業按時間間隔執行。
> 更多資訊：<https://crontab.guru/>.
```

File: pages.zh_TW/common/mysql.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+`mysql --socket {{sock檔路徑}}`
+
+- 執行腳本檔案(批次檔)中的SQL語句:
+
+`mysql -e "source {{sql檔案}}" {{資料庫名稱}}}`
+
+- 從使用`mysqldump`建立的備份還原資料庫 (系統將提示使用者輸入密碼):
+
+`mysql --user {{使用者名稱}} --password {{資料庫名稱}}} < {{sql備份檔路徑}}`
```
Comment (blueskyson) [inline pages.zh_TW/common/mysql.md:28]:
```suggestion
- 用 `mysqldump` 建立的備份還原資料庫（系統將提示使用者輸入密碼）：

`mysql --user {{使用者名稱}} --password {{資料庫名稱}} < {{sql 備份檔路徑}}`
```

**Review by blueskyson — COMMENTED** [review]
Thanks for contribution. Please modify copywriting with the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.zh.md). I give some suggestions, please apply the rules to all the pages.
The major suggestions are:
1. Put spaces between Chinese characters and half-width characters.
2. Use full-width punctuation unless a description's last word is English.

**Comment by Tylerastro** [comment]
@blueskyson 
Thank you for your suggestions. I've made changes accordingly.
There could be things I missed.
If there are any other aspects that still need attention please don't hesitate to let me know.

File: pages.zh_TW/common/aws.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+- 使用 SSO 設定 AWS 命令​​列：
+
+`aws configure sso`
+
+- 查看 AWS 指令​​的說明：
+
+`aws {{AWS指令}} help`
+
+- 取得呼叫者身分 (用於排除權限問題)：
```
Comment (blueskyson) [inline pages.zh_TW/common/aws.md:19]: Please use full-width parenthesis.
```suggestion
- 取得呼叫者身分（用於排除權限問題）：
```

**Review by blueskyson — APPROVED** [review]
Thanks!

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution and welcome to tldr.

**Pull request merged**